src/agent/debuglet.ts

```diff
@@ -597,6 +622,11 @@ export class Debuglet extends EventEmitter {
                 }
                 that.scheduleBreakpointFetch_(
                     that.config_.breakpointUpdateIntervalSec);
+                if (that.promise_) {
```

Comment:
For my comment above to work, this check should be `if (that.promiseResolve_)` ...

Reply:
Done.